### PR TITLE
libepoxy 1.5.5

### PR DESCRIPTION
**Pull request by tschoonj**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Review by chenrui333 — DISMISSED** [review]

**Comment by chenrui333** [comment]
```
env: python3: No such file or directory
```

File: Formula/libepoxy.rb [hunk 1]
```diff
@@ -25,6 +24,7 @@ class Libepoxy < Formula
   depends_on "meson" => :build
   depends_on "ninja" => :build
   depends_on "pkg-config" => :build
+  depends_on "python" => :build
```
Comment (chenrui333) [inline Formula/libepoxy.rb:27]: should we do `python@3.9` instead of `python`?

Comment (carlocab) [reply Formula/libepoxy.rb:27]: We should, as `python` is a moving target.

Comment (tschoonj) [reply Formula/libepoxy.rb]: I don't think it matters much, it just needs a `python3` interpreter at build-time.

**Review by carlocab — APPROVED** [review]

File: Formula/libepoxy.rb (hunk 1 above)
Comment (SMillerDev) [reply Formula/libepoxy.rb]: It matters when python migrates though

Comment (tschoonj) [reply Formula/libepoxy.rb]: But even then, the existing bottles will continue to work right? And afterwards when the package is rebuilt for whatever reason, the new version of python will be picked up automatically I assume?

**Pull request closed without merging**